google-assistant-sdk/googlesamples/assistant/library/hotword.py

```diff
@@ -16,7 +16,8 @@
 
 
 from __future__ import print_function
-
+import faulthandler
+faulthandler.enable()
```

Comment:
This seems to be failing the CI tests, as it's placed ahead of other imports. Can you move this to below all of the imports?